core/src/cpus/cortex_m.zig

```diff
@@ -684,6 +702,10 @@ pub const startup_logic = struct {
                 : .{ .memory = true, .r0 = true, .r1 = true });
         }
 
+        if (builtin.abi.float() == .hard) {
+            enable_fpu();
```

Comment:
I like that this change makes microzig have more "batteries included" by default, but let's allow the user to disable the FPU at build time, with a flag named `disable_fpu`.